docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md

```diff
@@ -105,7 +105,7 @@ The response follows the [standard format][], with a successful result containin
 | `ledger.close_time_resolution` | Number  | Ledger close times are rounded to within this many seconds. |
 | `ledger.closed`                | Boolean | Whether or not this ledger has been closed. |
 | `ledger.ledger_hash`           | String  | Unique identifying hash of the entire ledger. |
-| `ledger.ledger_index`          | String  | The [Ledger Index][] of this ledger, as a quoted integer. |
+| `ledger.ledger_index`          | [API v1][]: String <br> [API v2][]: Number  | The [Ledger Index][] of this ledger, as a quoted integer. |
```

Comment:
The "quoted" integer part in the description only applies to API v1. Can probably just scratch that entirely:

```suggestion
| `ledger.ledger_index`          | [API v1][]: String <br> [API v2][]: Number  | The [Ledger Index][] of this ledger. |
```